stable/consul/README.md

```diff
@@ -33,10 +33,11 @@ The following tables lists the configurable parameters of the consul chart and t
 | `Replicas`              | k8s statefulset replicas              | `3`                                                        |
 | `Component`             | k8s selector key                      | `consul`                                                   |
 | `Cpu`                   | container requested cpu               | `100m`                                                     |
-| `DisableHostNodeId`     | Disable Node Id creation (uses random)| `false`                                                   |
+| `DatacenterName`        | Consul Datacenter Name                | `dc1` (The consul default)                                 |
```

Comment:
Should we set a default if it's the same? Can you also add this to values.yaml even if it's commented out.